cdk-out/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "cdk-out",
+  "version": "0.1.0",
+  "bin": {
+    "cdk-out": "bin/cdk-out.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@types/jest": "^29.5.12",
+    "@types/node": "20.12.7",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.1.2",
+    "aws-cdk": "2.145.0",
+    "ts-node": "^10.9.2",
+    "typescript": "~5.4.5"
+  },
+  "dependencies": {
+    "aws-cdk-lib": "2.145.0",
+    "constructs": "^10.0.0",
+    "source-map-support": "^0.5.21",
+    "aws-cdk/core": "^2.0.0",
```

Comment:
I believe if you use "aws-cdk-lib" (CDK v2), then "aws-cdk/XXX" can be uninstalled (legacy CDK v1)